.pre-commit-config.yaml

```diff
@@ -38,3 +38,8 @@ repos:
         language: python
         additional_dependencies: [pygments, restructuredtext_lint]
         python_version: python3.6
+    -   id: changelogs-rst
+        name: changelog files must end in .rst
+        entry: ./scripts/fail
```

Comment:
since I think this will probably be an applicable pattern elsewhere, I think I'll make it a first-class thing: https://github.com/pre-commit/pre-commit/issues/807

Reply:
@asottile this misses the part that automatically renames them in the workdir

Reply:
yeah I left that as an exercise to the developer since making untracked files isn't usually a great idea (people often `git add -u && !!` which would untrack the file -- and generally touching the staging area seems not the best)